funcframework/framework.go

```diff
@@ -167,12 +182,6 @@ func handleEventFunction(w http.ResponseWriter, r *http.Request, fn interface{})
 		return
 	}
 
-	// Structured cloud events contain the context in the header, so we need to parse that out.
-	if isStructuredCloudEvent(r) {
-		runStructuredCloudEvent(w, r, body, fn)
-		return
-	}
-
 	// Legacy cloud events (e.g. pubsub) have data and an associated metadata, so parse those and run if present.
```

Comment:
These legacy events were never CloudEvents. Saying `cloud event` is confusing when referring to them.
We should do a quick search to replace `legacy cloud event` -> `legacy event` in the comments/errors in this repo with this change to disambiguate.